packages/p2p/src/components/buy-sell/buy-sell-modal.jsx

```diff
@@ -66,7 +66,11 @@ const BuySellModal = ({ table_type, selected_ad, should_show_popup, setShouldSho
         if (!should_show_popup) {
             setErrorMessage(null);
         }
-
+        buy_sell_store.getExchangeRate(
+            general_store.client.currency,
+            // general_store.client.local_currency_config.currency
```

Comment:
Could you please remove the commented code?

Reply:
Done, thx for notice